internal/controllers/clusterextension_controller.go

```diff
@@ -221,6 +228,15 @@ func SetDeprecationStatus(ext *ocv1alpha1.ClusterExtension, bundle *catalogmetad
 	}
 }
 
+func (r *ClusterExtensionReconciler) GenerateExpectedBundleSource(o ocv1alpha1.ClusterExtension, bundlePath string) *rukpakapi.BundleSource {
+	return &rukpakapi.BundleSource{
```

Comment:
So, this API is not a k8s API, it's just a struct as you point out. It was convenient to use. However, it's named `rukpakapi` since it's part of the interface to the pulled-in rukpak code.

Reply:
It had deepycopy methods, so thought this was intended to be a separate API. The naming is all good 👍